assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetGA4/indexMainDashboard.stories.js

```diff
@@ -0,0 +1,268 @@
+} from '../../../../analytics-4/utils/data-mock';
+import { CORE_USER } from '../../../../../googlesitekit/datastore/user/constants';
+import { getWidgetComponentProps } from '../../../../../googlesitekit/widgets/util';
+import { MODULES_ANALYTICS_4 } from '../../../../analytics-4/datastore/constants';
+import { replaceValuesInAnalytics4ReportWithZeroData } from '../../../../../../../.storybook/utils/zeroReports';
+import DashboardAllTrafficWidgetGA4 from '.';
+
+function limitResponseToSingleDate( analyticsResponse ) {
+	const findFirstDateRangeRow = ( dateRange ) =>
+		analyticsResponse.rows.find(
+			( { dimensionValues } ) => dimensionValues[ 1 ].value === dateRange
+		);
+
+	return {
+		...analyticsResponse,
+		rows: [
+			findFirstDateRangeRow( 'date_range_0' ),
+			findFirstDateRangeRow( 'date_range_1' ),
+		],
+	};
+}
+
+const widgetComponentProps = getWidgetComponentProps(
+	'analyticsAllTraffic-widget'
+);
+
+const Template = () => (
+	<DashboardAllTrafficWidgetGA4 { ...widgetComponentProps } />
+);
+
+const baseAllTrafficOptions = {
+	startDate: '2020-12-09',
+	endDate: '2021-01-05',
+	compareStartDate: '2020-11-11',
+	compareEndDate: '2020-12-08',
+	metrics: [
+		{
+			name: 'totalUsers',
+		},
+	],
+};
+
+const allTrafficReportOptions = [
+	{
+		...baseAllTrafficOptions,
+		dimensions: [ 'sessionDefaultChannelGrouping' ],
+		orderby: {
+			fieldName: 'totalUsers',
+			sortOrder: 'DESCENDING',
+		},
+		limit: 6,
+	},
+	{
+		...baseAllTrafficOptions,
+		dimensions: [ 'country' ],
+		orderby: {
+			fieldName: 'totalUsers',
+			sortOrder: 'DESCENDING',
+		},
+		limit: 6,
+	},
+	{
+		...baseAllTrafficOptions,
+		dimensions: [ 'deviceCategory' ],
+		orderby: {
+			fieldName: 'totalUsers',
+			sortOrder: 'DESCENDING',
+		},
+		limit: 6,
+	},
+	baseAllTrafficOptions,
+	{
+		startDate: '2020-12-09',
+		endDate: '2021-01-05',
+		dimensions: [ 'date' ],
+		metrics: [
+			{
+				name: 'totalUsers',
+			},
+		],
+	},
+];
+
+export const MainDashboardLoaded = Template.bind( {} );
+MainDashboardLoaded.storyName = 'Loaded';
+MainDashboardLoaded.args = {
+	setupRegistry: ( registry ) => {
+		allTrafficReportOptions.forEach( ( options ) => {
+			provideAnalytics4MockReport( registry, options );
+		} );
+	},
+};
+MainDashboardLoaded.scenario = {
+	label: 'Modules/Analytics/Widgets/DashboardAllTrafficWidgetGA4/MainDashboard/Loaded',
+};
+
+export const MainDashboardLoading = Template.bind( {} );
+MainDashboardLoading.storyName = 'Loading';
+MainDashboardLoading.args = {
+	setupRegistry: ( registry ) => {
+		allTrafficReportOptions.forEach( ( options ) => {
+			provideAnalytics4MockReport( registry, options );
+			registry
+				.dispatch( MODULES_ANALYTICS_4 )
+				.startResolution( 'getReport', [ options ] );
+		} );
+	},
+};
+MainDashboardLoading.decorators = [
+	( Story ) => {
+		// Ensure the animation is paused for VRT tests to correctly capture the loading state.
+		return (
+			<div className="googlesitekit-vrt-animation-paused">
```

Comment:
Without using the new `googlesitekit-vrt-animation-paused` class, this is how the Loading scenarios look:

![image](https://user-images.githubusercontent.com/18395600/216378133-fecc892f-e6c5-4c49-9c4f-548864364987.png)